_posts/blog/2019-08-19-Ruslan's-Final-Report.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: GSoC Final Report by Ruslan 
+categories: blog
+excerpt: work product submission on the scona project during GSoC program
+image:
+  feature:
+link:
+date: 2019-08-19
+modified:
+share: true
+author: ruslan_yermakov
+---
+
+## TL;DR - GSoC Summary
+
+> **Student**: Ruslan Yermakov  
```

Comment:
Should this whole block be quoted (using `>` syntax)? Seems like it will format strangely if only the first line is.

Or no quoting? The project part might be kinda long? Give rendering it a try and see what you think.

Reply:
Yep, rendering helps a lot to understand how the post will look like

Apparently, one `>` is enough to be quoted. 

Speaking about "The project part might be kinda long", it does not seem so to me, but  I made a second option

1) 
![image](https://user-images.githubusercontent.com/14000852/63106009-97855300-bf8a-11e9-9503-b655b25eb4fe.png)

2) 
![image](https://user-images.githubusercontent.com/14000852/63106045-b1269a80-bf8a-11e9-9dae-8af32ad55a89.png)

Which one do you prefer? :)

Reply:
I updated some gems because GitHub gave me a security warning, and now I need to add `<br>`s to each of those lines. I don't know why - some updated version of jekyll I presume - but they're in the PR I've opened to your master branch.

Reply:
Oh - and I like the second one which is the one you added I think - thank you!